api/src/project_view_history.ts

```diff
@@ -201,7 +204,6 @@ export function addHttpHandler(server: FastifyInstance, urlPrefix: string, servi
         for (const subproject of subprojects) {
           events.push(...subproject.log);
         }
-
```

Comment:
why? the lines above add the subproject log to the project log, while the line below sorts _all_ log events.. I'd keep the empty line to separate these two things